### ISSUE TITLE
Final UI: Initial entry-point

### ISSUE THREAD

**Issue by Siequnu** [cyipt/actdev-ui#11]
Here are 4 concepts that we have explored following our call. A separate issue will be created to discuss site-level visualisation and transition between these levels.

1. Geometric bubble-style top-level organisation, displayed in a non-geographical fashion, with size corresponding to a stat.
➕ Brings to the fore the key comparators
➕Vivid design
➖Non-geographic layout potentially disorientating for users
<img width="657" alt="Screenshot 2021-02-16 at 20 51 22" src="https://user-images.githubusercontent.com/11821606/108069081-eb9c5680-709d-11eb-8f3a-b103abfc94fc.png">

2. Standard map based approach, with geographic regions coloured to represent data.
➕ Familiarity for the part of the user base that understands maps
➖Very hard to get a large amount of information displayed at once sans popups, tooltips, other cluttering non-discoverable UI elements
➖The use of size to denote comparators would be problematic as they would heavily overlap 
<img width="525" alt="Screenshot 2021-02-16 at 20 51 19" src="https://user-images.githubusercontent.com/11821606/108069087-ec34ed00-709d-11eb-9faa-2866782a2ae9.png">

3. Carousel concept, sortable by a slider (or geographical area)?
➕ Intuitive navigation and data display
➕ Clear sense of exploration — feels like cards on table that you want to pick up and browse
➖The ordering is not intuitive, and adding additional sliders compounds this problem
➖Strongly needs a geolocation control (so that users can find "their" site), which is strongly at odds with the ordering concept
<img width="968" alt="Screenshot 2021-02-16 at 20 51 16" src="https://user-images.githubusercontent.com/11821606/108069062-e63f0c00-709d-11eb-873b-c53825af2b54.png">

4. Energy-efficiency lozenge style concept, giving vivid and clear readability and both regional and national level at-a-glance comparison.
➕ Enables users to compare sites at both a regional and national level.
➕ Enables top-level scenario control
➕ Reasonably easy to find your site, without typing or selecting a location (because sites are grouped by region)
➖Needs careful work to avoid clutter, but this is doable
<img width="930" alt="Screenshot 2021-02-16 at 21 16 53" src="https://user-images.githubusercontent.com/11821606/108071464-b9402880-70a0-11eb-89ed-521f5551fd86.png">

In our view, concept 4 seems the most promising. What are other people's thoughts?

**Comment by joeytalbot** [comment]
Looking good, I like number 4.

**Comment by mvl22** [comment]
I suggest we leave these concepts to mull for a little while, and move on to the next level that the site objects in the above would link through to.

@Siequnu, can you start a new issue for the next zoom level, as it were, and we can do a similar exercise on that?

I'd say as a guide overall we should be spending about 20% of the time on this entry point, and 80% on the detail level - as the latter is where the majority of user time is going to be spent, interrogating the data.

**Comment by Robinlovelace** [comment]
These are great ideas, excellent work. Of those options I would also say 4 is good - nice way of showing the different sites. 

As discussed with @mvl22 I would be happy with a simple interactive 'DataTable' approach that should be fast to implement - great JS: https://datatables.net/

Implementation: https://www.cyipt.bike/rapid/bedford/

Bog standard table that could quickly deliver what we need in minimal time:

![image](https://user-images.githubusercontent.com/1825120/108107960-da484f80-7087-11eb-8d6a-b9a0b03f1bf9.png)

More flashy version with colourful % bar (thinking % active under 2011 and active scenarios):

![image](https://user-images.githubusercontent.com/1825120/108108179-2e533400-7088-11eb-8abe-06dc65cce9d3.png)

Of course the n. columns and visuals could change and ultimately it's the data we're showing that's most important. I would say key variables to show or allow prioritisation are:

- [ ] % Active 2011 scenario
- [ ] % Active ActDev/Dutch scenario (or whatever we call it)
- [ ] Fast route busyness
- [ ] Quiet route busyness (for finding sites with no quiet routes)?

And possibly individual mode shares, e.g. walking/cycling/driving/PT.





> I suggest we leave these concepts to mull for a little while, and move on to the next level (i.e. issue #13) that the site objects in the above would link through to.

:+1:

**Comment by Siequnu** [comment]
Here is the latest iteration of the national-level zoom concept. We've brought in design concepts from the site-level zoom, and have aimed to achieve a high level of legibility and ease-of-use.

Your thoughts?

At this point we'd like to gain agreement of the team on the iterated concept in order to proceed to implementation.

<img width="1191" alt="Screenshot 2021-02-23 at 21 30 39" src="https://user-images.githubusercontent.com/11821606/108850619-878a0d00-761e-11eb-9566-0c6b6f0abdb5.png">

**Comment by Robinlovelace** [comment]
Looks fine to me but may be overkill for the purposes of this project. Tentative suggestion: create a simple table interface that works in minimal time (1 hr dev time?), get the main site level view to MVP status (~20 hrs dev time? appropriate given the relative importance of that element) and then revisit this after the MVP has been hit. I would also be happy if the MVP is implemented without any updates to the dropdown menu that's currently there.

I really do like the concept and hope to see it implemented, just not sure it's a good use of time at this critical phase when we need to be able to show the development version to stakeholders asap. On that note do you have a timeline for merging the dev branch into master? Will be good to see the new elements added iteratively with the auto-deploy.

**Comment by Siequnu** [comment]
I agree that the site-level zoom should be completed first. 

I expect by the start of next week to have finished incorporating the real data into the site-level mock-up — at which point it’ll be ready to merge into the master branch.

**Comment by Robinlovelace** [comment]
Sounds great, many thanks for preemptive comms on this. Got a good feeling about this, excited to see it come together and agree a visually informative and inviting landing page will help. Impressive if you can get the site-level aspect *and* this sorted by next week, looking forward to seeing how the updated site-level app looks and literally giving this site selection interface a 'spin' if there's time to implement it after the MVP is reached.
